Mathlib/AlgebraicTopology/SimplexCategory.lean

```diff
@@ -231,6 +231,10 @@ def mkOfLe {n} (i j : Fin (n+1)) (h : i ≤ j) : ([1] : SimplexCategory) ⟶ [n]
       | 0, 1, _ => h
   }
 
+@[simp]
+lemma mkOfLe_rfl_const {n} (j : Fin (n + 1)) :
+    mkOfLe j j (by omega) = [1].const [0] 0 ≫ [0].const [n] j := Hom.ext_one_left _ _
```

Comment:
I think the RHS would be better in this version:
```suggestion
@[simp]
lemma mkOfLe_refl {n} (j : Fin (n + 1)) :
    mkOfLe j j (by omega) = [1].const [n] j := Hom.ext_one_left _ _
```
(This lemma does not seem to be used in the main result here.)

Reply:
I've changed the lemma, as suggested, but the old version is used in the proof of the base case of fac_aux2 (see line 107) and I wasn't able to use the simpler version. Line 121 of the proof is 

```
refine congrArg (X.map ([1].const [0] 0).op) ?_
```
which uses the factorization of the constant map.